netbox/utilities/tests/test_filters.py

```diff
@@ -436,21 +469,21 @@ def test_site_slug_endswith_negation(self):
         params = {'slug__niew': ['-1']}
         self.assertEqual(SiteFilterSet(params, self.site_queryset).qs.count(), 2)
 
-    def test_site_asn_lt(self):
```

Comment:
The filterset no longer has asn mapping to the site model, instead it maps to the M2M.  If I change that, then we can leave it, otherwise I think we need to change it.

Reply:
If we're going to keep the `asn` field we should preserve its associated filters as well, otherwise it effects a breaking change.

Reply:
I will roll those back then as well.  I just thought it might be confusing if we have both ASNs and ASN.  I could also make a custom filter that checks both the ASN field as well as the ASN relationship. Thoughts on that?  Once we deprecate the site ASN field we can go to a more straight forward filter.

Reply:
I added back the ASN filters, however I feel like we should still move this filterset test off of ASN and onto something else in anticipation of removing ASNs.

If we don't want to use circuits, we could look at MTU on interfaces as I don't anticipate us ever changing that field so it should be a nice stable field.  I could just test against the ASN number in the new ASN model.